action.yml

```diff
@@ -20,10 +20,23 @@ inputs:
   private_api_key:
     description: ''
     required: true
+  working_dir: # used for s2i-core and s2i-base images only
```

Comment:
This is not needed. As 'core', 'base' for s2i-base-container as '1.22' for nginx-container are directories.

There is no difference. Why it this needed?

I don't follow.

Reply:
The main reason between core and base is the make command. When building base and core the VERSIONS seem to be empty. But I might be wrong?

Reply:
I rechecked and it seems, that what I wrote is not true.